src/index.html

```diff
@@ -1,13 +1,437 @@
+          <div class="header__top">
+            <a href="#menu" class="icon icon--menu"></a>
+
+            <a href="#home" class="logo">
+              <img
+                src="./images/logo.png"
+                alt="Miami Condo Kings logo"
+                class="logo__image"
+              >
+            </a>
+
+            <a href="tel:+1 234 555-55-55" class="icon icon--phone">
+              <div class="icon__tooltip">
+                +1 234 555-55-55
+              </div>
+            </a>
+          </div>
+
+          <h1 class="header__title">
+            Miami
+            <br>
+            Condo Kings
+          </h1>
+        </div>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top">
+            <a href="#" class="icon icon--cross"></a>
+
+            <a href="#" class="logo">
+              <img
+                src="./images/logo.png"
+                alt="Miami Condo Kings logo"
+                class="logo__image"
+              >
+            </a>
+          </div>
+
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#" class="menu__link">Home</a>
+            </li>
+            <li class="menu__item">
+              <a href="#about-us" class="menu__link">About us</a>
+            </li>
+            <li class="menu__item">
+              <a href="#gallery" class="menu__link">Gallery</a>
+            </li>
+            <li class="menu__item">
+              <a href="#testimonials" class="menu__link">Testimonials</a>
+            </li>
+            <li class="menu__item">
+              <a href="#contacts" class="menu__link">Contacts</a>
+            </li>
+          </ul>
+
+          <a href="tel:+1 234 555-55-55" class="menu__call menu__call--phone">
+            +1 234 555-55-55
+          </a>
+          <a href="tel:+1 234 555-55-55" class="menu__call menu__call--action">
+            call to order
+          </a>
+        </div>
+      </div>
+    </nav>
+
+    <main>
+      <section class="page__section about" id="about-us">
+        <div class="container">
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--desktop-1-4">
+              <h2 class="about__title">
+                Miami best
+                <br>
+                Real estate
+
+                <a href="#" class="link">here</a>
+              </h2>
+            </div>
+
+            <div class="grid__item grid__item--desktop-7-12">
+              <a href="#" class="about__subtitle link">
+                Over 50 years experience in love.
+              </a>
+
+              <p class="about__paragraph">
+                Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.
+              </p>
```

Comment:
move text to a new line, try to keep your code without horizontal scroll, fix in all places